.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea

```

Comment:
These IDE specific ignores should be defined in your local repo only, e.g.
- .git/info/exclude
- or using a [global excludes file](https://help.github.com/articles/ignoring-files)

Reply:
> These IDE specific ignores should be defined in your local repo only

Which guide/reference/best practise are you following here (I am curious)? Jetbrains actually encourages to commit .idea files (which I don't like as this is a environment agnostic project). Atlassian mentions IDE-configurations as one of the things that should go into a project .gitignore. Ros also mentions 'Please note that folder .idea with project files will be created. You can add it to .gitignore file in case if you are using Git and do not want to commit this file to repository.'

What kind of harm does it do to have that defined on a project level? This way I can clone the repository wherever I want without having to configure every machine individually. I'd actually encourage collaborators to add their IDE-files (KDevelop, Eclipse, Emacs) to the .gitignore as well.
However, this might be my person preference. If there is a strong objection against project configurations in `.gitignore`, we can remove them and I have to handle my different settings some other way.